tools/release_builder.py

```diff
@@ -44,7 +45,30 @@ def get_package_name(stub_universe_url: str) -> str:
                 )
 
             package_name = name_match.group(1)
-            log.info("Got package name %s from stub universe URL")
+
+            try:
+                universe_name = None
+                log.info("Parsing stub universe...")
+                with urllib.request.urlopen(stub_universe_url) as response:
+                    data = json.loads(response.read())
+                    for package in data["packages"]:
+                        if "name" in package:
+                            if universe_name is not None and universe_name != package["name"]:
+                                log.warning(
+                                    "Two different package names have been found '{}' and '{}'".format(
+                                        universe_name, package["name"]
+                                    )
+                                )
+                            else:
+                                universe_name = package["name"]
+                if universe_name is not None:
+                    package_name = universe_name
+            except urllib.error.HTTPError:
+                log.warning("Could not open URL: {}".format(stub_universe_url))
```

Comment:
Please do not hide the details provided in the original exception. This makes troubleshooting very hard - we'd have no idea whether the problem was transient, permanent, server-side, client-side, bad proxy or what.